saw-remote-api/src/SAWServer/ProofScript.hs

```diff
@@ -62,13 +81,29 @@ instance FromJSON Prover where
   parseJSON =
     withObject "prover" $ \o -> do
       (name :: String) <- o .: "name"
+      let unints = fromMaybe [] <$> o .:? "uninterpreted functions"
       case name of
-        "abc"   -> pure ABC
-        "cvc4"  -> CVC4 <$> o .: "uninterpreted functions"
-        "rme"   -> pure RME
-        "yices" -> Yices <$> o .: "uninterpreted functions"
-        "z3"    -> Z3 <$> o .: "uninterpreted functions"
-        _       -> empty
+        "abc"            -> pure ABC_Internal
+        "internal-abc"   -> pure ABC_Internal
```

Comment:
Nitpick: alphabetize these `case` alternatives to make it easier to tell if it's in sync with the documentation.

Reply:
:+1: